docs/docs/CMDB/FAQ.md

```diff
@@ -0,0 +1,209 @@
+# FAQ
+
+## Why not use existing models from NetBox?
+
+[NetBox](https://netbox.dev) is a great tool to manage your DCIM and IPAM.
+
+Our CMDB models directly use the DCIM and IPAM models, like `Device` and `IP prefixes`.
+
+NetBox also includes several models which could be considered as CMDB such as `VLAN`, `VRF` and `L2VPN`. We do not want to use these models to avoid confusion and design mismatches with our CMDB.
+
+## Why not implement directly OpenConfig models in the CMDB?
+
+OpenConfig models are designed from a device perspective.
+
+Our CMDB models are designed from a service/asset perspective. It enables us to apply common parameters without worrying of applying them at both ends.
+
+Example: we can set a maintenance status on a BGP session directly.
+
+It also enables us to limit redefinition of values that can lead to misconfiguration. For instance, in OpenConfig nothing prevents us from having a mismatch of ASN in a BGP session. In our CMDB, it cannot happen as the BGP session is the central object which links two devices, hence the neighbor A remote-as is the neighbor B local-as.
```

Comment:
It also enables us to limit redefinition of values that can lead to misconfiguration. ->
It also enables us to prevent conflicting device configurations.